### DISCUSSION TITLE
How to createValidator using authz module?

### DISCUSSION THREAD

**Discussion by Mrigesh901** [cosmos/community#64] [Validators, Nodes, Relayers, etc...] [answered]
I wanted to use authz module in coreum chain, My use case is i want to authorize grantee to run createValidator command and also use grantee's  tokens. I have already created the tx of granter,grantee pair using cored tx authz grant command. But i don't know what to do after that such that createValidator command can be run by grantee's account. Please help me with this.

**Comment by atheeshp** [comment]
Hey, you need to generate unsigned txn (using --generate-only flag) from granter and broadcast (using `exec` command) it from grantee.
https://github.com/cosmos/cosmos-sdk/blob/0ba62db727799f333c6c58ba63397f4322bb7afe/x/authz/README.md#msgexec
https://github.com/cosmos/cosmos-sdk/blob/0ba62db727799f333c6c58ba63397f4322bb7afe/x/authz/module/autocli.go#L57

**Reply by Mrigesh901** [reply]
Hey, thank you. In the next step, after the unsigned transaction is created and saved as tx.json, how do we send it to grantee side. Is there a method by which grantee can broadcast the transaction from any UI wallet like keplr or metamask . For example, grantee does not want to run cli, and has a browser wallet, how does he get tx.json to his wallet from the backend and he just have to approve it.

**Reply by atheeshp** [reply, marked as answer]
> how do we send it to grantee side

Hey, you don't need to send txn to grantee, the grantee can create the unsigned txn with the address of granter directly
for example:
```
simd tx bank send granterAddr cosmos1... 10stake --generate-only > tx.json
```

> Is there a method by which grantee can broadcast the transaction from any UI wallet like keplr or metamask

https://resolute.vitwit.com is having UI for some authz operations, but I guess `createValidator` is not included (may be upcoming)
